testapp/src/main/java/com/rakuten/tech/mobile/testapp/legacy/MainActivity.kt

```diff
@@ -49,7 +49,7 @@ class MainActivity : AppCompatActivity() {
     }
 
     companion object {
-        const val APP_ID = "b9247b09-c534-48c5-bcc5-0644907d22f8"
-        const val VERSION_ID = "9761a1dc-a1cb-4730-92ac-e9aa09a87531"
+        const val APP_ID = "d005410f-4055-4d95-a0c9-899a60c5936a"
```

Comment:
This is in legacy code. This should not be referred to and shouldn't require any modification.